deployments/anvilprod/environment.py

```diff
@@ -987,7 +987,45 @@ def mkdict(previous_catalog: dict[str, str],
+    bqsrc('bf0b5c71', 'ccdg_asc_ndd_daly_talkowski_ac_boston_asd_exome_20250507_ANV5_202505080038'),
+    bqsrc('de55c8ca', 'ccdg_asc_ndd_daly_talkowski_aleksic_asd_exome_20250514_ANV5_202505191339'),
+    bqsrc('05f81c53', 'ccdg_asc_ndd_daly_talkowski_barbosa_asd_exome_20250514_ANV5_202505191347'),
+    bqsrc('ec751e53', 'ccdg_asc_ndd_daly_talkowski_brusco_asd_exome_20250514_ANV5_202505191354'),
+    bqsrc('e880eb24', 'ccdg_asc_ndd_daly_talkowski_chung_asd_exome_20250514_ANV5_202505191404'),
+    bqsrc('14218b2f', 'ccdg_asc_ndd_daly_talkowski_control_NIMH_asd_exome_20250514_ANV5_202505191412'),
+    bqsrc('44dd3f7f', 'ccdg_asc_ndd_daly_talkowski_domenici_asd_exome_20250514_ANV5_202505191423'),
+    bqsrc('b497e3bd', 'ccdg_asc_ndd_daly_talkowski_gargus_asd_exome_20250514_ANV5_202505191431'),
+    bqsrc('23e0cf66', 'ccdg_asc_ndd_daly_talkowski_goethe_asd_exome_20250514_ANV5_202505191439'),
+    bqsrc('dae963c2', 'ccdg_asc_ndd_daly_talkowski_gurrieri_asd_exome_20250514_ANV5_202505191448'),
+    bqsrc('02a4023f', 'ccdg_asc_ndd_daly_talkowski_herman_asd_exome_20250514_ANV5_202505191456'),
+    bqsrc('562d7351', 'ccdg_asc_ndd_daly_talkowski_hertz_picciotto_asd_exome_20250514_ANV5_202505191503'),
+    bqsrc('90758277', 'ccdg_asc_ndd_daly_talkowski_hertz_picciotto_asd_wgs_20250514_ANV5_202505191512'),
+    bqsrc('cd6d2aa6', 'ccdg_asc_ndd_daly_talkowski_hultman_asd_exome_20250514_ANV5_202505191519'),
+    bqsrc('d21464d2', 'ccdg_asc_ndd_daly_talkowski_kolevzon_asd_exome_20250514_ANV5_202505191537'),
+    bqsrc('186b5498', 'ccdg_asc_ndd_daly_talkowski_kolevzon_asd_wgs_20250514_ANV5_202505191546'),
+    bqsrc('7fc43dc3', 'ccdg_asc_ndd_daly_talkowski_lattig_asd_exome_20250514_ANV5_202505191554'),
+    bqsrc('e4ab9e05', 'ccdg_asc_ndd_daly_talkowski_mayo_asd_exome_20250515_ANV5_202505191602'),
+    bqsrc('81d3c9c8', 'ccdg_asc_ndd_daly_talkowski_mcpartland_asd_exome_20250515_ANV5_202505191610'),
+    bqsrc('0b020eb2', 'ccdg_asc_ndd_daly_talkowski_menashe_asd_exome_20250515_ANV5_202505191620'),
+    bqsrc('f49b0d4c', 'ccdg_asc_ndd_daly_talkowski_minshew_asd_exome_20250515_ANV5_202505191632'),
+    bqsrc('d0e9181a', 'ccdg_asc_ndd_daly_talkowski_parellada_asd_exome_20250515_ANV5_202505191640'),
+    bqsrc('3965a084', 'ccdg_asc_ndd_daly_talkowski_passos_bueno_asd_exome_20250515_ANV5_202505191651'),
+    bqsrc('c9d758bb', 'ccdg_asc_ndd_daly_talkowski_pericak_vance_asd_exome__20250515_ANV5_202505191658'),
+    bqsrc('45f811c7', 'ccdg_asc_ndd_daly_talkowski_persico_asd_exome_20250515_ANV5_202505191706'),
+    bqsrc('c3a807a3', 'ccdg_asc_ndd_daly_talkowski_renieri_asd_exome_20250515_ANV5_202505191714'),
+    bqsrc('d610c1c1', 'ccdg_asc_ndd_daly_talkowski_schloesser_asd_exome_20250515_ANV5_202505191722'),
+    bqsrc('ea8f0099', 'ccdg_asc_ndd_daly_talkowski_weiss_asd_exome_20250515_ANV5_202505191738'),
+    bqsrc('f143f633', 'ccdg_broad_ai_ibd_daly_mcgovern_share_wes_20240104_ANV5_202503171541'),
+]))
+
+anvil11_sources = mkdict(anvil10_sources, 372, mkdelta([
```

Comment:
You'll need to pop `AnVIL_ENCORE_RS293_20250304_ANV5_202503042020` in `anvil11` to make the count match.

Reply:
`AnVIL_ENCORE_RS293_20250304_ANV5_202503042020` is replaced in `anvil11` with `ANVIL_ENCORE_RS293_20250710_ANV5_202507211507`.

I found the cause, some projects where replaced (the popped ones) with lowercase instead of capital letters which Azul was still detecting as new.